qiskit/qpy/interface.py

```diff
@@ -210,6 +222,14 @@ def dump(
             annotation_factories=annotation_factories,
         )
 
+    if version >= 16:
+        # Write the byte offsets for each program
+        file_obj.seek(table_start)
+        for offset in byte_offsets:
+            file_obj.write(struct.pack(formats.CIRCUIT_TABLE_ENTRY_PACK, offset))
```

Comment:
Just to be consistent, you might want to do this:

```suggestion
            file_obj.write(struct.pack(formats.CIRCUIT_TABLE_ENTRY_PACK, *formats.CIRCUIT_TABLE_ENTRY(offset) ))
```

Reply:
Thank you, I've commit the suggested change.